src/c20_client/do_wait.py

```diff
@@ -0,0 +1,20 @@
+import time
+
+# WAITING_TIME_FOR_EACH_CALL = (60 mins * 60 seconds)/1000 = 3.6
+# 0.06s additional time for buffering
+
+WAITING_TIME_FOR_EACH_CALL = 3.66
+
+
+def get_wait_time():
+    sleep_time = WAITING_TIME_FOR_EACH_CALL
+    return sleep_time
+
+
+def wait():
+    time.sleep(get_wait_time())
+
+
+def wait_between_jobs():
+    get_wait_time()
+    wait()
```

Comment:
`wait_between_jobs` calls `get_wait_time` but does nothing with its return value.  Then, in the call to `wait`, `get_wait_time` is called a second time.  This second call uses the return value to call `sleep`.

This has an extra layer of functions that should be removed.

Reply:
There are no tests for this.